astropy/time/core.py

```diff
@@ -897,14 +897,72 @@ def jd2(self):
         jd2 = self._time.mask_if_needed(self._time.jd2)
         return self._shaped_like_input(jd2)
 
-    @property
-    def value(self):
-        """Time value(s) in current format"""
-        # The underlying way to get the time values for the current format is:
-        #     self._shaped_like_input(self._time.to_value(parent=self))
-        # This is done in __getattr__.  By calling getattr(self, self.format)
-        # the ``value`` attribute is cached.
-        return getattr(self, self.format)
+    def to_value(self, format=None, subfmt=None):
```

Comment:
@taldcroft - in an earlier comment you suggested that one should be forced to give a `format` (maybe for `TimeDelta`, though). I think that may be worth considering: it makes it very explicit what will be given back. (Obviously, in that case `value` has to be turned back into an explicit property)